src/main/java/frc/robot/commands/intake/SpinIntakeCommand.java

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands.intake;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.Intake;
+
+/**
+ * Repeatedly spins the {@link Intake} at a given speed.
+ */
+public class SpinIntakeCommand extends CommandBase {
+
+  public final Intake intake;
+  public final double speed;
+
+  /**
+   * Creates a command that spins the {@link Intake} at a given speed.
+   *
+   * @param intake The {@link Intake} to control
+   * @param speed The speed at which to spin the intake
+   */
+  public SpinIntakeCommand(Intake intake, double speed) {
```

Comment:
Should this command also have a constructor that takes a supplier for the speed?

Reply:
the ticket said "takes a double (not a supplier)" but idk maybe?

Reply:
Oh. Missed that, sorry!